src/server/index.js

```diff
@@ -0,0 +1,64 @@
+import http from 'http';
+import url from 'url';
+import { db } from './orientdb';
+import { TweetBuilder } from '../shared/data/tweet';
+import { TweeterBuilder } from '../shared/data/tweeter';
+import { flattenImmutableObject } from '../shared/utilities';
+
+const port = process.env.PORT || 3000;
+
+const server = http.createServer((request, response) => {
+  if (request.method === 'GET') {
+    const queryData = url.parse(request.url, true).query;
+    response.writeHead(200, { 'Content-Type': 'text/plain' });
+
+    db.query('SELECT FROM tweet WHERE @rid="#14:1"')
+      .then((tweetRecords) => {
+        for (const tweetRecord of tweetRecords)  {
+          const tweeterId = tweetRecord.tweeter;
+          console.log('TWEET FINDING')
+          console.log('' + tweetRecord['@rid'])
+
+
+          db.query('TRAVERSE in FROM (SELECT in("TWEETED") from :id)',
+            {'params':{'id': tweetRecord['@rid']}}
+        )
+            .then((tweeterRecord_) => {
+              const tweeterRecord = tweeterRecord_[0];
+              console.log("THEN")

```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Mixed double and single quotes.](https://www.codacy.com/app/winwardo/solid-octo-disco/file/2164912261/issues/source?bid=3148782&fileBranchId=3160401#l28)